include/umf/memory_pool_ops.h

```diff
@@ -105,7 +106,7 @@ typedef struct umf_memory_pool_ops_t {
     ///         Whether any status other than UMF_RESULT_SUCCESS can be returned
     ///         depends on the memory provider used by the \p pool.
     ///
-    umf_result_t (*free)(void *pool, void *);
+    umf_result_t (*free)(void *pool, void *ptr);
```

Comment:
add "[out]" to @param ptr - please keep this consistent across all public API

Reply:
it is not out :D